jekyll/_docs/1.0/status-badges.md

```diff
@@ -1,55 +0,0 @@
----
-layout: classic-docs
```

Comment:
we still have 1.0 users, so we must keep this document for now.

Reply:
I agree, but in this case, it's a duplicate article. There is a replica located at `jekyll/_cci1/status-badges.md`. The only contents of `_docs` are this duplicate article.

Reply:
People would still look for it. Especially for direct links. I would have kept it and redirected the 1.0 version to the new one so that there aren't any 404s.

Reply:
Oh, I see, I think that old page is not published on the 1.0 site, so it should be ok to delete it

Reply:
It is published, which is good. Searching "CircleCI status badges" on Google returns the 1.0 version of the docs page so we don't want that to 404. :)